### PR TITLE
craftos-pc: 2.8 -> 2.8.1

### PR DESCRIPTION
**Pull request by tomodachi94**
## Description of changes

Changelog: https://github.com/MCJack123/craftos2/releases/tag/v2.8.1

This contains a bunch of bug fixes that were released in v2.8.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/applications/emulators/craftos-pc/default.nix
Comment (OPNA2608) [inline pkgs/applications/emulators/craftos-pc/default.nix:23]: Any reason we can't use 2.8.1 release?
https://github.com/MCJack123/craftos2-lua/releases/tag/v2.8.1

File: pkgs/applications/emulators/craftos-pc/default.nix
Comment (OPNA2608) [inline pkgs/applications/emulators/craftos-pc/default.nix:29]: Any reason we can't use 2.8.1 release?
https://github.com/MCJack123/craftos2-rom/releases/tag/v2.8.1

Comment (tomodachi94) [reply pkgs/applications/emulators/craftos-pc/default.nix]: Oops, the developer mentioned there was no v2.8.1 update for an Accelerated (other version of the software) package so I assumed this was the same. Fixed in latest push.

File: pkgs/applications/emulators/craftos-pc/default.nix
Comment (tomodachi94) [reply pkgs/applications/emulators/craftos-pc/default.nix]: Oops, the developer mentioned there was no v2.8.1 update for an Accelerated (other version of the software) package so I assumed this was the same. Fixed in latest push.

**Review by OPNA2608 — APPROVED** [review]
Result of `nixpkgs-review pr 283733` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>craftos-pc</li>
  </ul>
</details>

---

Bildschirmfoto_2024-01-28_21-43-25

LGTM. Do you want to wait for a review from the co-maintainer?

**Comment by tomodachi94** [comment]
> LGTM. Do you want to wait for a review from the co-maintainer?

I'm not really sure. I've had PRs for this package merged without his approval before, but I don't know what the convention in Nixpkgs is.

**Comment by tomodachi94** [comment]
Oops 🙈

**Comment by tomodachi94** [comment]
I guess let's just go for it @OPNA2608, I've done package updates without a review from the other maintainer(s) before and nobody's complained yet.

**Pull request merged**
